backend/de.metas.postfinance/src/main/java/de/metas/postfinance/document/export/IPostFinanceYbInvoiceHandler.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * #%L
+ * de.metas.postfinance
+ * %%
+ * Copyright (C) 2024 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.postfinance.document.export;
+
+import javax.annotation.Nullable;
+
+public interface IPostFinanceYbInvoiceHandler
+{
+	boolean applies (final PostFinanceYbInvoiceRequest postFinanceYbInvoiceRequest);
```

Comment:
imo the parameters should all be `@NonNull` (the 2 methods here and also the implementations)